.circleci/config.yml

```diff
@@ -297,3 +342,22 @@ workflows:
               only: /.*/
             branches:
               only: main
+
+      - sync-dags-branch:
+          name: 🔃 Update DAGs branch
+          filters:
+            tags:
+              only: /.*/
+            branches:
+              only: main
+
+      - sync-dags-repo:
+          name: 🔃 Synchronize telemetry-airflow-dags repository
+          filters:
+            tags:
+              only: /.*/
```

Comment:
same comment as above. My assumption is that on commits to main, these branches will be updated so the sidecars can sync them

Reply:
Fixed as well